submissions/nauvian-perspective-bentham/nauvian-perspective-bentham.md

```diff
@@ -0,0 +1,46 @@
+
+Over the course of 2014, Bentham began a few different Factorio series. In August 2014, he began his popular and most favourite [Factorio Railworld](https://www.youtube.com/watch?v=aw9zgzrF2DY&list=PLOx2-nvzE_ZI69s1psHkAdeH2_uNmP9rg&index=1) series and the [Factorio No Crafting Challenge](https://www.youtube.com/watch?v=FTszEGCARW4&list=PLOx2-nvzE_ZKIrTTzJ5etReO2iFeo_WIp) series. Each video in the Railworld series attracted tens of thousands of views which grew his channel from a few hundred subscribers to many thousands. 
+
+For some reason, I always had it in my mind that Bentham was the first person who properly coined the idea of a "main bus", although at the time he called it the "tree method", as the bus was the trunk with production lines on the sides as the branches. [He said in the interview](audio clip) that in the forums, the "main bus" phrase started cropping up, and it took him a while to realise that it was the same thing as his tree method. Of course it's likely that the main bus is just the natural evolution of many Factorio players that wish to leave the spaghetti playstyle. 
+
+If you a long-time Factorio fan, you may have heard of him from his very popular [Factorio Rail Tutorials](https://youtu.be/XSGYSbEPpbM) which provided a much-needed handy guide to understanding and using trains and signals. I remember not getting over that first steep signals learning curve until I watched these videos. He also did a video on initial oil too, as that also used to be difficult to pick up first time.
+
+Later, in 2015, he began my absolute favourite Bentham series, [Living with Biters](https://www.youtube.com/watch?v=83KSIH4dJg8&list=PLOx2-nvzE_ZJXS1sQDUzWtvBLn9ZPyUXk). The idea of living amongst biter bases on peaceful mode is still fascinating to me, as it poses multiple challenges you wouldn’t get in any other normal vanilla campaign. This base is so interesting that stringweasel plans to put it into his fantastic [Hall of Fame mod](https://mods.factorio.com/mod/HallOfFame).
+
+It was around mid-2015 when he reached the monumental mid-point of his journey around the colossal lake, in his Railworld series. Coincidentally, at the same time, the 0.15 update dropped, bringing the Railworld preset. Bentham wondered if maybe the idea was from his Railworld series, so I asked Klonan (he created the preset) about it, who said "I want to keep some mystery in the world", so, we'll leave that thought at that :)*maybe remove this paragraph? seems like a nice thing to attribute to Bentham but then again, Klonan's answer...*
```

Comment:
Well, it's actually _really_ annoying that Klonan said that because after I finished recording in the interview, Bentham told me that Klonan (or someone else, he couldn't remember) had told him that the name of the preset was inspired by his Factorio series, when he had asked about it. Since I didn't have that comment recorded, I decided to ask Klonan, who gave me this answer. It may be for weird legal reasons, or just Klonan being Klonan, but it does put me in an odd situation. Now I can't really ask Bentham to type that out or say for an audio clip, as Klonan could see that as me trying to go against his will. A sticky situation, just for something I really wanted to attribute to him as having a large and positive affect on the game.